prime-router/metadata/fhir_mapping/hl7/message/ORU_R01.yml

```diff
@@ -61,6 +61,21 @@ resources:
       - .OBSERVATION.NTE
       - MSH
 
+  - resourceName: ServiceRequest
```

Comment:
This had to be pulled out so that all the OBX segments from the specimen were passed in

Reply:
All of the reference documents from Carlos (like [this one](https://lucid.app/lucidchart/dc67748e-b67e-4224-80e6-27e26d6359c1/edit?invitationId=inv_5f0d1789-cbf2-43b5-b0b1-adc762fecd08&page=0_0#)) show that ServiceRequests only exist within a Diagnostic Report. Don't know if those documents are more of a reflection of the way we currently lay things out, or if it was tied to the FHIR specs. It may just be that we've never had input messages that necessitated this kind of change and that this is all good, but I haven't done my research really, so was wondering if you had any thoughts on that chart based on what you've looked into.

Reply:
So even before this change the ServiceRequest was still an object in resources rather than something nested in DiagnosticReport and it was referenced by DiagnosticReport, this change just means that we make it before rather than having it be created when we make the diagnostic report.

I can leave a comment though that makes that more obvious

Reply:
Ah okay that makes sense, I don't think a comment is needed, I think I just need to look more at the actual resulting bundle instead of just looking at changed files.